README.md

```diff
@@ -27,29 +31,33 @@ Or simply from the online master branch, which is stable:
 pip install git+https://github.com/pylhc/PyLHC.git#egg=pylhc
 ```
 
-After installing, codes can be run with either `python -m pylhc.SCRIPT --FLAG ARGUMENT` or calling path to the `.py` file directly.
+After installing, scripts can be run with either `python -m pylhc.SCRIPT --FLAG ARGUMENT` or by calling the `.py` files directly.
 
 Note: some of the scripts access functionality only available on the CERN Technical Network.
 To use those, you should make sure to install the relevant extra dependencies with `pip install path/to/Pylhc[tech]`.
 
 ## Description
 
-This package provides tools which can be useful for working with accelerators, but are not neccessary for optics measurements analysis.
+This package provides tools useful in our workflows working on accelerator physical, but that are not neccessary for optics measurements analysis.
```

Comment:
that is a weird sentence. work work physical

Reply:
This package provides tools for particle accelerator data analysis and to extract machine information, complementing the optics measurement analysis tools of the [omc3 package](https://github.com/pylhc/omc3).

Reply:
Made a slight change, let me know what you think